File: Cap 10/Topico 10.7/verdados/verifdados.cpp
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 10 - Topico 10.7
 *
 * Programa ..: OPERACAO COM SUBROTINA DE FUNCAO
 */

#include <iostream>
using namespace std;

bool IGUAL(int A, int B) {
  return A == B;
}

int X, Y;

int main(void) {
  cout << "VERIFICACAO DE DADOS" << endl;
  cout << "FUNCAO SIMPLES COM PASSAGEM DE PARAMETRO" << endl << endl;
  cout << "Informe o 1o. valor: ";
  cin >> X;
  cout << "Informe o 2o. valor: ";
  cin >> Y;
  cout << endl;
  if (IGUAL(X, Y))
    cout << "Valores sao iguais" << endl;
  else
    cout << "Valores sao diferentes" << endl;
  return 0;
}

File: Cap 10/Topico 10.7/verdados/verifdados.js
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 10 - Topico 10.7
 * 
 * Programa ..: OPERACAO COM SUBROTINA DE FUNCAO
 */

const readline = require('readline-sync');

function IGUAL(A, B) {
  return A === B;
}

let X, Y;
console.log("VERIFICACAO DE DADOS");
console.log("FUNCAO SIMPLES COM PASSAGEM DE PARAMETRO\n");
X = parseInt(readline.question("Informe o 1o. valor: "));
Y = parseInt(readline.question("Informe o 2o. valor: "));
console.log();
if (IGUAL(X, Y)) {
  console.log("Valores sao iguais");
} else {
  console.log("Valores sao diferentes");
}
